src/main/resources/db/data/V2__insert_data.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE Categories(
```

Comment:
The name of the file is "insert_data" but you are not inserting any data, you just create the tables. Please do a refactor here